pkg/nsx/services/vpc/vpc.go

```diff
@@ -970,3 +973,49 @@ func (s *VPCService) GetNSXLBSPath(lbsId string) string {
 	}
 	return *vpcLBS.Path
 }
+
+func GetAlbEndpoint(cluster *nsx.Cluster) error {
+	_, err := cluster.HttpGet(albEndpointPath)
+	return err
+}
+
+func (vpcService *VPCService) NSXLBEnabled() bool {
+	lbProviderMutex.Lock()
+	defer lbProviderMutex.Unlock()
+
+	if lbProvider == "" {
+		lbProvider = vpcService.getLBProvider()
+	}
+	return lbProvider == common.NSX_LB
+}
+
+func (vpcService *VPCService) getLBProvider() string {
```

Comment:
Can we rename it like UseAVILoadBalancer, so the return value will always be true or false, no need to define constant type names

Reply:
it copied the NCP implement.
I have moved definition from common to vpc internal since only NSXLBEnabled  provided. There is no need to define constant type names

Reply:
and is it possible that we could other lb ? use type name inside vpc seems ok.